tests/test_maudau.py

```diff
@@ -0,0 +1,127 @@
+import datetime as DT
+import json
+import os
+import pytest
+import tempfile
+from pyspark.sql import SparkSession
+from pyspark.sql.types import StructField, StructType, StringType
+from mozetl.maudau import maudau as M
+
+NARROW_SCHEMA = StructType([
+    StructField("client_id",             StringType(),  True),
+    StructField("submission_date_s3",    StringType(),  False),
+    StructField("subsession_start_date", StringType(), True)])
+
+generated = M.format_as_submission_date(M.today.isoformat())
+
+spark = (
```

Comment:
There is a shared spark fixture with the same name in `conftest.py`. You can use it in any test function or fixture by adding it as a function parameter.